### PR TITLE
Contest submission backend

### PR DESCRIPTION
**Pull request by ridhishjain**

### CODE REVIEW

File: backend/Models/contests/subjective/updateSubjective.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const { pool } = require('../../db');
 
 function updateSubjective({ contestId, questionId, username, score }) {
   return new Promise((resolve, reject) => {
-    pool,
+    pool.
```
Comment (nveenjain) [inline backend/Models/contests/subjective/updateSubjective.js]: use the queryDB library provided previously...

File: backend/Routes/Controllers/contests/subjective/index.js [hunk 2]
```diff
@@ -3,18 +3,23 @@ const ajv = require('../../../../Schema');
...
+const validator = require('../../../../Schema');
 const {
   checkSchema,
   updateSchema
 } = require('../../../../Schema/contests/subjective');
 
 router.get('/:adminid/:contestid/:username/:questionid', async (req, res) => {
-  const validate = ajv.compile(checkSchema);
-  const valid = validate(req.body);
+  let validate = validator.compile(checkSchema.checkSubjective);
```
Comment (nveenjain) [inline backend/Routes/Controllers/contests/subjective/index.js]: Please use the responseMiddleware provided here.

File: backend/Routes/Controllers/contests/submission/index.js [hunk 3]
```diff
@@ -51,12 +52,16 @@ router.get('/:contestid/:username/result', async (req, res) => {
...
 
 router.post('/:contestid/:username/:questionid/submit', (req, res) => {
-  let validate = ajv.compile(submitSchema);
+  let validate = validator.compile(submitSchema);
   let valid = validate(req.body);
   if (!valid) {
     return res.status(400).json({
       sucess: false,
-      error: ValidityState.errors,
+      error: validate.errors.reduce
```
Comment (nveenjain) [inline backend/Routes/Controllers/contests/submission/index.js]: Use response middleware here..

**Comment by AnuragGupta93** [comment]
PR can be merged now.
@nveenjain Sir

**Pull request merged**